packages/worker/worker.js

```diff
@@ -79,23 +80,24 @@ const runScript = async ({ script, url, base=globalBase, root=base }) => {
   const fromSource = getParameterDefinitionsFromSource(script)
   def = combineParameterDefinitions(fromSource, await scriptModule.getParameterDefinitions?.())
   main = scriptModule.main
-  await runMain({ params: extractDefaults(def) })
-  return {def}
+  let out = await runMain({ params: extractDefaults(def) })
+  out.def = def
+  return out
 }
 
 // TODO remove, or move to another package, along with exportStlText
 // this is interesting in regards to exporting to stl, and 3mf which actually need vertex data, 
-// and not jcad geometry polygons. So it will be interesting to see if main can give back transferable buffers
+// and not jcad geometry polygons. So it will be interesting to can give back transferable buffers
 // instead of re-running conversion. or move export to main thread where the data already is, as it is needed for rendering
 const exportData = async (params) => {
-  if(self.exportData) return self.exportData(params, solids, entities)
+  if(self.exportData) return self.exportData(params)
 
   const { format } = params
   // todo check if it is ok to give back transferables after webgl has used the buffers
   // then we would not need to clone the data
   // other option is to clone data before sending transferable
   JscadToCommon.clearCache()
-  if (solids.length && !entities.length) entities = JscadToCommon(solids, [], false)
+  let entities = JscadToCommon(solids, [], false)
```

Comment:
`entities` can be `const` here?

Reply:
I have a nasty habbit of not caring about const :D let is shorter, and const optimizations are for hot code in my opinion at least. I agree it is a good habbit to start with const, but if you do not mind too much, I would leave let :D :D :D :D.